sdcm/cluster_k8s/eks.py

```diff
@@ -41,6 +43,7 @@
 )
 from sdcm.remote import LOCALRUNNER
 from sdcm.utils.aws_utils import (
+    EksClusterCleanupMixin,
```

Comment:
Previous commit mistakenly removed the import of the `EksClusterCleanupMixin` class.
Now it is added...